packages/qgrid-core/.eslintrc.json

```diff
@@ -0,0 +1,64 @@
+{
+    "env": {
+        "browser": true,
```

Comment:
check indentation through this file.
Seems like it's tabs in there. Make indentation with 2 spaces

Reply:
Tabs are everywhere in this commit.